### ISSUE TITLE
Integration with ScalaCheck's test runner doesn't work

### ISSUE THREAD

**Issue by Bobo1239** [lihaoyi/mill#286]
I'm trying to use Mill with ScalaCheck's own test runner (complete example [here](https://github.com/Bobo1239/mill-scalacheck))
```scala
object foo extends ScalaModule {
  def scalaVersion = "2.12.4"
  object test extends Tests {
    def ivyDeps        = Agg(ivy"org.scalacheck::scalacheck:1.13.5")
    def testFrameworks = Seq("org.scalacheck.ScalaCheckFramework")
  }
}
```
but running `mill foo.test` doesn't run any tests. @rockjam investigated this issue and concluded that it's a ScalaCheck-specific problem.

**Comment by nsadeh** [comment]
Hello, can I get a quick explanation for this? I see it was closed, is there a resolution?

**Comment by lefou** [comment]
It was closed with a commit (d94bb5eae6cd5f75d3cac1b69518b80c0a90169b), so I think the issue was fixed.
